### ISSUE TITLE
Narrator does not announce the selected changing state for the tab items (Request Body etc.)

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1148]
**Test Environment:**
OS Build: Windows 11
Browser Version: Microsoft Edge Dev Version 92.0.891.1
Screen Reader: Narrator
URL: Graph Explorer - Microsoft Graph

**Repro steps:**
1. Open above URL on Edge Dev browser.
2. Turn on the Narrator 'Ctrl+Win+Enter' keys
3. Navigate to 'Request Body' tab key and right arrow key and select it by enter/space key.
4. Observe the Narrator's narration

**Actual Result:**
Narrator does not announce the selected changing state for the tab items (Request Headers etc.), also it announce the name of the tab items twice
It announces as 'Request Body request body tab item 1 of 4' and after activating the tab item, it again announces as ' 'Request Body request body tab item 1 of 4'

**Expected Result:**
Narrator should not announce the name of the tab item twice, and after selecting the tab item, Narrator should announce the changing state.
It should announce as 'Request body tab item 1 of 4' and after activating the tab item, it should announce as 'Selected'

**User Experience:**
Narrator users will be impacted here as they will not know that the tab item has got selected or not and also they will be confused with the repetitive announcement of the control

**Observation:**
1. Issue does not reproes with NVDA screen reader
2. Issue reproes on scan mode as well

**NOTE:**
1. Issue reproes for all the tab item on the page

**Comment by ElinorW** [comment]
This issue is only reproducible on **Windows 11**
